### PR TITLE
Added New OCP Project

### PR DESCRIPTION
**Pull request by stefanborcia**

### CODE REVIEW

File: OCP_Example1/Chef.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OCP_Example1
+{
+    public  class Chef : Employee
+    {
+        public Chef(string firstName, string lastName, string department, int salary) : base(firstName, lastName, department, salary)
+        {
+
```
Comment (GabrielCorlan) [inline OCP_Example1/Chef.cs:12]: you should remove empty spaces

File: OCP_Example1/Employee.cs [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+        public abstract void PerformDuties();
+
+        public override string ToString()
+        {
+            return
+                $"The Employee with Id: {ID}, First name: {FirstName} , Last name: {LastName}, the Deparment: {Department}, has a salary of {Salary}";
+        }
+    }
+
```
Comment (GabrielCorlan) [inline OCP_Example1/Employee.cs]: same as above

File: OCP_Example1/Janitor.cs [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OCP_Example1
+{
+    public class Janitor : Employee
+    {
+        public Janitor(string firstName, string lastName, string department, int salary) : base(firstName, lastName, department, salary)
+        {
+
```
Comment (GabrielCorlan) [inline OCP_Example1/Janitor.cs]: same as above

File: OCP_Example1/RestaurantManager.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    {
+        // This method should call any employee to do his job
+        public void CallEmployee(Employee employee)
+        {
+            employee.PerformDuties();
+        }
+
+       
+
+
```
Comment (GabrielCorlan) [inline OCP_Example1/RestaurantManager.cs]: same as above

File: OCP_Example1/Security.cs [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OCP_Example1
+{
+    public class Security : Employee
+    {
+        public Security(string firstName, string lastName, string department, int salary) : base(firstName, lastName, department, salary)
+        {
+
```
Comment (GabrielCorlan) [inline OCP_Example1/Security.cs]: same

File: OCP_Example1/Waiter.cs [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OCP_Example1
+{
+    public class Waiter : Employee
+    {
+        public Waiter(string firstName, string lastName, string department, int salary) : base(firstName, lastName, department, salary)
+        {
+
```
Comment (GabrielCorlan) [inline OCP_Example1/Waiter.cs:12]: same

**Review by GabrielCorlan — CHANGES_REQUESTED** [review]

File: OCP_Example1/Chef.cs (hunk 1 above)
Comment (stefanborcia) [reply OCP_Example1/Chef.cs:12]: Done

File: OCP_Example1/Employee.cs (hunk 2 above)
Comment (stefanborcia) [reply OCP_Example1/Employee.cs]: Done

File: OCP_Example1/RestaurantManager.cs (hunk 4 above)
Comment (stefanborcia) [reply OCP_Example1/RestaurantManager.cs]: Done

File: OCP_Example1/Security.cs (hunk 5 above)
Comment (stefanborcia) [reply OCP_Example1/Security.cs]: Done

File: OCP_Example1/Waiter.cs (hunk 6 above)
Comment (stefanborcia) [reply OCP_Example1/Waiter.cs:12]: Done

File: OCP_Example1/Janitor.cs (hunk 3 above)
Comment (stefanborcia) [reply OCP_Example1/Janitor.cs]: Done

**Review by GabrielCorlan — APPROVED** [review]
LGTM

**Pull request merged**